RELEASENOTES.md

```diff
@@ -1 +1 @@
-
+- Add `wiat-for-migration` command. By default it waits for all ongoing migrations to finish and reports the number of all in progress and queued ones in each query cycle. If an optional `migration-id` is provided, it will additionally wait for that specific migration to finish.
```

Comment:
```suggestion
- Add `wait-for-migration` command. By default it waits for all ongoing migrations to finish and reports the number of in progress and queued migrations. If an optional `migration-id` is provided, it will return as soon as that migration is complete.
```

Reply:
addressed in dc1a57c0c399b2dc339ff6fcbae5058e0adae071

Reply:
addressed in dc1a57c0c399b2dc339ff6fcbae5058e0adae071